modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -972,7 +987,7 @@ protected function register_controls() {
 			Group_Control_Box_Shadow::get_type(),
 			[
 				'name' => 'box_shadow_box_shadow',
-				'selector' => "{$nested_tabs_content_selector_class}",
+				'selector' => '{{WRAPPER}} .e-n-tabs-content',
```

Comment:
This is incorrect. Now the styling will be applied to child nested tab widgets as well.

Reply:
Probably missed it in the merging from 3.10.0.
Fixed